src/puppetlabs/i18n/main.clj

```diff
@@ -17,10 +17,24 @@
   ;; Very simple localization
   (println (trs "Welcome! This is localized"))
 
+  ;; Very simple plural system localization
+  (println (trs "{0} of beer on the wall."
+                (trsn "One bottle" "{0} bottles" 99)))
+
   ;; String with arguments
-  (let [nprog 3 nmonths 5]
+  (let [nprog 3
+        nmonths 5]
     (println (i18n/tru "It took {0} programmers {1} months to implement this"
-                      nprog nmonths)))
+                       nprog nmonths)))
+
+  ;; String with arguments
+  (let [variants [[1 1] [2 2] [3 5] [7 1] [1 7]]]
+    (doseq [[nprog nmonths] variants]
+
+      ; first item is number of programmers, second is number of months
+      (println (str (i18n/tru "It took {0} {1} to implement this"
+                              (i18n/trun "one programmer" "{0} programmers" nprog)
+                              (i18n/trun "one month" "{0} months" nmonths))))))
```

Comment:
@LarissaLane This is the example we were talking about the other day in the meeting, of say two items that may need pluralization in a single sentence.  I think this is a good (if potentially the only?) way to solve this problem.  Is this clear to you (and or other translators?) Would it require further comment/context in an actual application?